buzzmoon-frontend/buzzmoon-vite/src/Components/CreateQuestion/CreateQuestion.jsx

```diff
@@ -2,27 +2,93 @@ import * as React from 'react';
 import './CreateQuestion.css';
 
 export default function CreateQuestion(props) {
+  const [questionAudio, setQuestionAudio] = React.useState();
+  const [playingMode, setPlayingMode] = React.useState('no audio');
+  // options are "no audio", "playing", "paused"
+
+  React.useEffect(() => {
+    if (props.question.audioFile) {
+      const newAudio = new Audio(URL.createObjectURL(props.question.audioFile));
+      newAudio.onended = () => { setPlayingMode('paused'); };
+      setQuestionAudio(newAudio);
+    }
+  }, [props.question.audioFile]);
+
+  const handleUploadFile = (e) => {
```

Comment:
It seems like your handler is assuming your file is coming in the right format. But what happens if they upload a PDF file? an image?